php/libraries/FeedbackMRI.class.inc

```diff
@@ -70,7 +70,7 @@ class FeedbackMRI
     function clearAllComments(): bool
     {
         // create DB object
-        $DB =& Database::singleton();
+        $DB = NDB_Factory::singleton()->database();
```

Comment:
missing "\", it is better to change 
$DB = NDB_Factory::singleton()->database(); 
to
$DB = \NDB_Factory::singleton()->database();